src/domain/comments/commentsSubscription.graphql

```diff
@@ -0,0 +1,11 @@
+subscription Comments($commentsId: UUID!) {
```

Comment:
Can you rename this to the appropriate subscription name? This does not mean much this way

Reply:
It's a subscription for receiving brand new messages.